QModManager/Patcher.cs

```
@@ -123,15 +123,23 @@ internal static void LoadMods()
                         {
                             try
                             {
-                                if (Newtonsoft.Json.Linq.JObject.Parse(File.ReadAllText(modjson[0].FullName))
-                                    .TryGetValue("Enabled", out Newtonsoft.Json.Linq.JToken isEnabled) &&
-                                    !(bool)isEnabled)
+                                JObject jObject = JObject.Parse(File.ReadAllText(modjson[0].FullName));
+                                if (
+                                    (jObject.TryGetValue("Enabled", out JToken isEnabled) || jObject.TryGetValue("Enable", out isEnabled))
+                                    && !(bool)isEnabled
+                                )
                                 {
                                     Console.WriteLine("Cannot resolve Assembly " + dll.Name + " - Disabled by mod.json");
                                     continue;
                                 }
+                                else
+                                {
+                                    Console.WriteLine($"Found dll: {dll.Name}, IS ENABLED");
+                                }
+                            }
+                            catch { /*fail silently*/
+                                Console.WriteLine($"FAILED to parse config for dll {dll.Name}");
```

No longer failing silently, can remove redundant comment